src/mesh/index_derivs.cxx

```diff
@@ -233,6 +233,8 @@ BoutReal FDDX_C4(stencil &v, stencil &f) {
 /// Non-oscillatory, containing No free parameters and Dissipative (NND) scheme
 /// http://arxiv.org/abs/1010.4135v1
 BoutReal FDDX_NND(stencil &v, stencil &f) {
+  throw BoutException("This converges only first order.\n"
```

Comment:
What's the expected order (I couldn't see this in a skim of the reference)? Aren't limiter methods like this a combination of method orders, dropping back to first order near steep gradients to ensure we don't introduce extra oscillations?

Reply:
I tested only smooth functions (sin,cos) - and I thought it should be higher.
If you say that first order for smooth functions is the expected behaviour - I am happy to remove the warning.

Reply:
I don't really know what the expected behaviour is for this specific operator.

Reply:
I think this function can be removed; it was added as a test many many years ago, and never really used for anything. Perhaps it can be added again later if tested

Reply:
I thought keeping it wouldn't hurt ...
If somebody was using it, this way they get a clear error message, and it is easy to re-enable, if this works for that use case ... but I have no strong opinion - I would also be happy to remove it

Reply:
I'm with @bendudson on this. If it was experimental in the first place and we're not certain that it works, let's remove it entirely for now. We can put it back if we get it working/tested.